docs/eportal/README.md

```diff
@@ -1136,46 +993,44 @@ an HTTP balancer.
 3. On the `eportal1` define `NODE_URL` and `REPLICATION_SHARED_KEY` in the
    configuration file:
 
-   ```
-   NODE_URL = 'http://eportal1.corp'
-   REPLICATION_SHARED_KEY = 'secret'
-   ```
+```text
+NODE_URL = 'http://eportal1.corp'
+REPLICATION_SHARED_KEY = 'secret'
+```
 
 4. On the `eportal2` define `NODE_URL`, `LEADER_URL` and `REPLICATION_SHARED_KEY`:
 
-   ```
-   NODE_URL = 'http://eportal2.corp'
-   LEADER_URL = 'http://eportal1.corp'
-   REPLICATION_SHARED_KEY = 'secret'
-   ```
+```text
+NODE_URL = 'http://eportal2.corp'
+LEADER_URL = 'http://eportal1.corp'
+REPLICATION_SHARED_KEY = 'secret'
+```
 
 5. That's it. After ePortal restart on both hosts, changes on the `eportal1`
    will be replicated to the `eportal2` instance and vice versa.
 
 :::tip Note
 You can change the KernelCare agent settings to point to a cluster hostname via:
+
+```text
+# curl -s http://eportal.corp/set-patch-server | bash
 ```
-curl -s http://eportal.corp/set-patch-server | bash
-```
+
 :::
 
 ### Adding node to an existing ePortal instance
 
-If you already have a working ePortal instance, you can setup a second node,
-define `NODE_URL` and `LEADER_URL` on both instances and trigger full sync on
-the follower instance:
+If you already have a working ePortal instance, you can setup a second node, define `NODE_URL` and `LEADER_URL` on both instances and trigger full sync on the follower instance:
 
-```
+```text
 [root@eportal2.corp]$ kc.eportal replication --full-sync
 ```
 
 ### Monitoring
 
-You can use `kc.eportal replication --short-status` CLI command to check replication
-status. It exits with code `1` if there are some issues with replication and
-outputs JSON:
+You can use `kc.eportal replication --short-status` CLI command to check replication status. It exits with code `1` if there are some issues with replication and outputs JSON:
 
-```
+```json
```

Comment:
json highlighting is kinda broken and looks barely visible:
![image](https://github.com/cloudlinux/tuxcare-documentation/assets/31970/a0e1bccc-b602-4363-8772-063a724888ff)
Maybe it's better to stick to text for now.

Reply:
yes i'll fix that - python and json come out a bit dark, i'll change to `text`